README.md

```diff
@@ -19,6 +19,8 @@ NumPyro is designed to be *lightweight* and focuses on providing a flexible subs
 
 ## Installation
 
+> **Limited Windows Support:** Note that NumPyro is untested on Windows, and will require building jaxlib from source. See https://github.com/google/jax/issues/438 for more details. 
```

Comment:
nit: trailing space appears here. It might be better to hide the link such as " For more details, see [this discussion](https://github.com/google/jax/issues/438).".

Reply:
Yeah, doesn't seem like github links get shortened in the readme. Done.